npm/ux/estimatesOverview.tsx

```diff
@@ -179,60 +193,64 @@ export function EstimatesOverview(props: {
   if (props.isSimplifiedView) {
     return (
       <>
-        <ResultsTable
-          columnNames={columnNames}
-          rows={props.estimatesData.map((dataItem, index) =>
-            reDataToRow(dataItem, colormap[index]),
-          )}
-          initialColumns={initialColumns}
-          ensureSelected={true}
-          onRowDeleted={props.onRowDeleted}
-          selectedRow={selectedRow}
-          setSelectedRow={onRowSelected}
-        />
-        <ScatterChart
-          xAxis={xAxis}
-          yAxis={yAxis}
-          data={props.estimatesData.map((dataItem, index) =>
-            reDataToRowScatter(dataItem, colormap[index]),
-          )}
-          onPointSelected={onPointSelected}
-        />
+        <g id="qs-estimatesOverview">
+          <ResultsTable
+            columnNames={columnNames}
+            rows={props.estimatesData.map((dataItem, index) =>
+              reDataToRow(dataItem, colormap[index]),
+            )}
+            initialColumns={initialColumns}
+            ensureSelected={true}
+            onRowDeleted={props.onRowDeleted}
+            selectedRow={selectedRow}
+            onRowSelected={onRowSelected}
+          />
+          <ScatterChart
+            xAxis={xAxis}
+            yAxis={yAxis}
+            data={props.estimatesData.map((dataItem, index) =>
+              reDataToRowScatter(dataItem, colormap[index]),
+            )}
+            onPointSelected={onPointSelected}
+          />
+        </g>
       </>
     );
   }
 
   return (
     <>
-      <details open>
-        <summary style="font-size: 1.5em; font-weight: bold; margin: 24px 8px;">
-          Results
-        </summary>
-        <ResultsTable
-          columnNames={columnNames}
-          rows={props.estimatesData.map((dataItem, index) =>
-            reDataToRow(dataItem, colormap[index]),
-          )}
-          initialColumns={initialColumns}
-          selectedRow={selectedRow}
-          setSelectedRow={onRowSelected}
-          ensureSelected={true}
-          onRowDeleted={props.onRowDeleted}
-        />
-      </details>
-      <details open>
-        <summary style="font-size: 1.5em; font-weight: bold; margin: 24px 8px;">
-          Space-time diagram
-        </summary>
-        <ScatterChart
-          xAxis={xAxis}
-          yAxis={yAxis}
-          data={props.estimatesData.map((dataItem, index) =>
-            reDataToRowScatter(dataItem, colormap[index]),
-          )}
-          onPointSelected={onPointSelected}
-        />
-      </details>
+      <g id="qs-estimatesOverview">
```

Comment:
`g` is an SVG element, so you shouldn't be using it in an HTML context such as here. You should probably just make the parent `<></>` element be a `div` with the id you want. (Though if using it for selection, you should use a className rather than id to find it in case there are multiple on the page)